### PR TITLE
Fix previous PR: exec.CommandContext cannot be reused

### PR DESCRIPTION
**Pull request by Gulio**
Changed logic to recreate exec.CommandContext when retrying with a new flag

### CODE REVIEW

File: packages/apt_deb.go [hunk 1]
```diff
@@ -213,10 +215,10 @@ func InstallAptPackages(ctx context.Context, pkgs []string) error {
 			cmd.Env = append(os.Environ(), "DEBIAN_FRONTEND=noninteractive")
 		},
 	}
-	stdout, stderr, err := runAptGet(ctx, args, cmdModifiers)
+	stdout, stderr, err := runAptGetWithDowngradeRetrial(ctx, args, cmdModifiers)
```
[the remark this answers is not in the document]
Comment (Gulio) [reply packages/apt_deb.go:218]: Done, I left runAptGetWithDowngradeRetrial in apt-get upgrade and apt-get install methods

**Review by adjackura — APPROVED** [review]

**Pull request merged**